buildbot/buildbot/config_default.py

```diff
@@ -34,14 +34,19 @@
 CONFIG_OPTIONS = {
     'skipseashell': bool,
     'estimate': bool,
+    'directives': str,
     'make': bool,
+    'kernel': str,
     'sdsflags': str,
     'platform': str,
 }
 
 # The name to use for compiled executables.
 EXECUTABLE_NAME = 'sdsoc'
 
+# Default directives file is empty.
+DEFAULT_DIRECTIVES = ''
```

Comment:
I don't think we need this—this is the only sensible default for `directives`; it wouldn't make sense for this to be nonempty.